obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+- agasc_id:
+  - 989727248
+  comments: UNKNOWN
+  mp_starcat_time: 2019:092:13:40:47.000
+  status: 1
+- agasc_id:
+  - 188501784
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2019:113:10:12:41.647
+  status: 1
+- agasc_id:
+  - 120457376
+  comments: Faint star affected by imposters. Set magnitude to MAXMAG
+  mp_starcat_time: 2019:121:21:47:55.999
+  status: 1
+- agasc_id:
+  - 1105998760
+  comments: Star gets 2 mags dimmer
+  mp_starcat_time: 2019:190:08:02:57.107
+  status: 1
+- agasc_id:
+  - 836381648
+  comments: Faint star. Affected by imposters. Set mag to MAXMAG
+  mp_starcat_time: 2019:245:18:05:01.870
+  status: 1
+- agasc_id:
+  - 157951008
+  comments: Faint star. Tracking affected by imposters. Marginally acceptable
+  mp_starcat_time: 2019:304:07:33:22.655
+  status: 0
+- agasc_id:
+  - 166993856
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2019:336:01:10:48.142
+  status: 1
+- agasc_id:
+  - 1178221128
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2019:336:03:32:40.722
+  status: 1
+- agasc_id:
+  - 122159368
+  comments: Affected by imposters
+  mp_starcat_time: 2019:357:18:41:21.840
+  status: 0
+- agasc_id:
+  - 1108353064
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:026:14:02:58.573
+  status: 1
+- agasc_id:
+  - 33690776
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:045:02:16:37.894
+  status: 1
+- agasc_id:
+  - 110493968
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:055:11:57:17.800
+  status: 1
+- agasc_id:
+  - 33690776
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:057:21:03:53.096
+  status: 1
+- agasc_id:
+  - 111028544
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:059:19:29:07.754
+  status: 1
+- agasc_id:
+  - 187829568
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:060:11:59:00.012
+  status: 1
+- agasc_id:
+  - 33690776
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:063:06:09:44.589
+  status: 1
+- agasc_id:
+  - 33690776
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:063:09:21:38.457
+  status: 1
+- agasc_id:
+  - 798760200
+  comments: Never acquired; set mag 11.2 +/- 0.1
+  mp_starcat_time: 2020:083:13:35:57.577
+  status: 1
+- agasc_id:
+  - 553396016
+  comments: High Background
+  mp_starcat_time: 2020:096:13:24:30.943
+  status: 1
+- agasc_id:
+  - 891159184
+  comments: Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2020:139:21:23:16.181
+  status: 1
+- agasc_id:
+  - 891159184
+  comments: Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2020:140:00:46:15.767
+  status: 1
+- agasc_id:
+  - 694561568
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:228:13:19:56.547
+  status: 1
+- agasc_id:
+  - 396501520
+  comments: Never acquired; set mag to MAXMAG
+  mp_starcat_time: 2020:240:12:46:53.718
```

Comment:
I don't see this star in the catalog for 46955 or 46950 so this seems odd?

Reply:
This is OBSID 46955 and I do see it here: https://icxc.harvard.edu/mp/mplogs/2020/AUG2420/oflsa/starcheck.html#obsid46955

Reply:
Gotcha.  This is an ER that ran after an observing-only interrupt, so the obsid was re-used and it gets a little confusing (the mica search by obsid shows a different catalog that is also right but for a different time and not relevant).  So this seems fine.